### PR TITLE
feat: improve MR options by adding flags and arguments

### LINKED ISSUE #14: feat: add "remove branch on merge" on MR creation

### PR DESCRIPTION
**Pull request by ryshu**
- Add flag to keep source branch when created MR while be merged
- Add flag to squash commits when created MR while be merged
- Add `assignee_ids` argument to `ensure_mr` cmd to allow user to set
some assignee_ids by default on created MR
- Add `reviewer_ids` argument to `ensure_mr` cmd to allow user to set
some reviewer_ids by default on created MR

Closes #14 , #15

### CODE REVIEW

**Comment by leorolland** [comment]
Looks ok but haven't tested it in a real Gitlab instance.

File: gitflow_toolbox/ensure_mr.py [hunk 1]
```diff
@@ -5,31 +5,73 @@
...
 @click.command()
-@click.option("--remote/--current", default=False)
 @click.argument("source_branch", type=str)
 @click.argument("target_branch", type=str)
 @click.argument("title", type=str)
 @click.argument("description", type=str)
 @click.argument("labels", type=str, nargs=-1)
+@click.option(
+    "--keep-source-branch",
+    "-ksb",
```
Comment (leorolland) [inline gitflow_toolbox/ensure_mr.py:15]: Use a single letter as [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
```suggestion
    "-k",
```

File: gitflow_toolbox/ensure_mr.py [hunk 2]
```diff
@@ -5,31 +5,73 @@
...
+@click.option(
+    "--remote/--current",
+    "-r/-c",
+    default=False,
+    help="Flag which allow you to chose between configured target or remote to run this tool."
+    " By default, the script use --current flag.",
+)
+@click.option(
+    "--assignee_ids",
+    "-ai",
```
Comment (leorolland) [inline gitflow_toolbox/ensure_mr.py:35]: Use a single letter as [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
```suggestion
    "-a",
```

File: gitflow_toolbox/ensure_mr.py [hunk 3]
```diff
@@ -5,31 +5,73 @@
...
+    type=int,
+    multiple=True,
+    help="Flag (accepting multiple values) used to specify the assignee IDs to set while creating the merge request.",
+)
+@click.option(
+    "--reviewer_ids",
+    "-ri",
```
Comment (leorolland) [inline gitflow_toolbox/ensure_mr.py:42]: Use a single letter as [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
```suggestion
    "-r",
```

**Review by leorolland — CHANGES_REQUESTED** [review]

File: gitflow_toolbox/ensure_mr.py (hunk 3 above)
Comment (ryshu) [reply gitflow_toolbox/ensure_mr.py:42]: It's to prevent conflict with `-r` for `--remote` flag

File: gitflow_toolbox/ensure_mr.py (hunk 2 above)
Comment (ryshu) [reply gitflow_toolbox/ensure_mr.py:35]: To keep same logic as reviewer_ids

**Pull request merged**